doc/level1.rst

```diff
@@ -928,12 +928,12 @@ import it to enable Autocrypt.  If the user agrees to do so:
    If there is no ``Passphrase-Format`` header, or the value is unknown,
    then the MUA MUST provide a plain UTF-8 string text entry.
 
- * The MUA should try decrypting the message with the supplied
-   Setup Code.  The Code serves both for decryption as well as
+ * The MUA should try decrypting the message with the supplied Setup
+   Code.  The Code serves both for decryption as well as
    authenticating the message.  Extra care needs to be taken with some
    PGP implementations that the Setup Code is actually used for
-   decryption. See :doc:`bad-import` for more explanation and an
-   example.
+   decryption. For example, `this is quite tricky to do correctly with
```

Comment:
tricky? :)

Reply:
yep, tricky -- we could use the word "difficult" instead if we want more formal language.

Reply:
amended to use "difficult" instead of "quite tricky"  -- thanks for the language check!